Makefile

```diff
@@ -131,55 +131,54 @@ build-all: build-kyverno build-kyvernopre build-cli ## Build all
 # BUILD (KO) #
 ##############
 
-INITC_KIND_IMAGE    := kind.local/github.com/kyverno/kyverno/cmd/initcontainer
-KYVERNO_KIND_IMAGE  := kind.local/github.com/kyverno/kyverno/cmd/kyverno
+INITC_KIND_IMAGE    := ko.local/github.com/kyverno/kyverno/cmd/initcontainer
+KYVERNO_KIND_IMAGE  := ko.local/github.com/kyverno/kyverno/cmd/kyverno
 INITC_IMAGE         := kyvernopre
 KO_PLATFORM         := linux/amd64,linux/arm64,linux/s390x
 REPO_KYVERNO        := $(REPO)/kyverno
 REPO_KYVERNOPRE     := $(REPO)/kyvernopre
 REPO_CLI            := $(REPO)/kyverno-cli
+REGISTRY_USERNAME	?= dummy
+
+.PHONY: ko-login
+ko-login: $(KO)
+	@$(KO) login $(REGISTRY) --username $(REGISTRY_USERNAME) --password $(REGISTRY_PASSWORD)
 
 .PHONY: ko-build-initContainer
-ko-build-initContainer: $(KO)
-	# @$(KO) login $(REGISTRY)
+ko-build-initContainer: ko-login
 	@LD_FLAGS=$(LD_FLAGS) KO_DOCKER_REPO=$(REPO_KYVERNOPRE) $(KO) build $(KYVERNOPRE_DIR) --bare --tags=latest,$(IMAGE_TAG) --platform=$(KO_PLATFORM)
 
 .PHONY: ko-build-kyverno
-ko-build-kyverno: $(KO)
-	# @$(KO) login $(REGISTRY)
+ko-build-kyverno: ko-login
 	@LD_FLAGS=$(LD_FLAGS) KO_DOCKER_REPO=$(REPO_KYVERNO) $(KO) build $(KYVERNO_DIR) --bare --tags=latest,$(IMAGE_TAG) --platform=$(KO_PLATFORM)
 
 .PHONY: ko-build-cli
-ko-build-cli: $(KO)
-	# @$(KO) login $(REGISTRY)
+ko-build-cli: ko-login
 	@LD_FLAGS=$(LD_FLAGS) KO_DOCKER_REPO=$(REPO_CLI) $(KO) build $(CLI_DIR) --bare --tags=latest,$(IMAGE_TAG) --platform=$(KO_PLATFORM)
 
 .PHONY: ko-build-initContainer-dev
-ko-build-initContainer-dev: $(KO)
-	@$(KO) login $(REGISTRY) --username "dummy" --password $(GITHUB_TOKEN)
+ko-build-initContainer-dev: ko-login
 	@LD_FLAGS=$(LD_FLAGS_DEV) KO_DOCKER_REPO=$(REPO_KYVERNOPRE) $(KO) build $(KYVERNOPRE_DIR) --bare --tags=latest,$(IMAGE_TAG_DEV) --platform=$(KO_PLATFORM)
 
 .PHONY: ko-build-kyverno-dev
-ko-build-kyverno-dev: $(KO)
-	@$(KO) login $(REGISTRY) --username "dummy" --password $(GITHUB_TOKEN)
+ko-build-kyverno-dev: ko-login
```

Comment:
Do we need to login every time we build locally, there might be cases that dev do not want login ?

Reply:
They can use `ko-build-kyverno-local` ?

Reply:
right, we can use